domains/pixel-host.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Bossa9973",
+           "email": "",
+           "discord": "821118817925201960",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.FOiVlKRjHg943CpeEqgHjBgOKozGL_-M75Mqam_p682o34yGMa0sdE2jvwvE5xDfVsSfNCG9uwmvbBz5uCg3upgP1bSo4RoHRxNaCRVdorgvnTAGL6sMlXuFqSZMsH1JDu4Dc4oXULjftpMElmSU8Qud7eBLbBvDS3UUNIei8PmUSjywtrPDQ8_J-OgeITWQWMHza1Bs6QzijZHg-5Xrl5Cfa88hmki8nxevIcxZWNJm7FTqmShguskmKDmOlhiJ_uONOUvZ4Kx8q3ADYeH35TXpxbx4ooJRpd4-ZIETnUq1T7XAwZckx2J08AKQ7ZIPjO5H6Fa--f2aK36d7fKi9g.uy9FP-5pSZP-IYTHs-ZFwQ.88gj2Yqdqr4943z-k_f6WjlVYXfxyZWTW8uytBoztY3iEoCWpsVS30TV1E37VRgaUcPQB3nD6gj1QfIO3XQJ_BqMojx-R8jJ8TUQPApFh1M.15FBrWUgjXgGVCqBJnKp-w"
+        },
+    
+        "record": {
+            "A": ["10.2.153.97"]
```

Comment:
Unable to access your site, are you able to send a preview? Also provide some reasoning for this domain.